packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1226,7 +1226,12 @@ describe.skipIf(flakey)('build', () => {
 
       const files = await fs.readdir(output);
       // we should NOT see `functions` because that means `middleware.ts` was processed
-      expect(files.sort()).toEqual(['builds.json', 'config.json', 'static']);
+      expect(files.sort()).toEqual([
+        'builds.json',
+        'config.json',
+        'diagnostics',
```

Comment:
should we check the contents of `diagnostics`?

Reply:
I've added a expectation for this afterwards 👌